app/views/layouts/application.html.slim

```diff
@@ -12,7 +12,7 @@ html
           .main_container
             = render :partial => 'layouts/header'
```

Comment:
@GauravMakwana-BTC : Update the syntax like = render partial:  'layouts/header', update this in all places